scripts/ci_checks/android/macos.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
```

Comment:
nit: I prefer to split different types of options (with and without arguments) for more readability: `set -eux -o pipefail`

Reply:
```
git grep 'set -euxo pipefail' | wc -l
25
```

but this is the style happened to be used in this project